README.md

```diff
@@ -19,6 +19,7 @@ UMF comes with predefined pool allocators (see include/pool) and providers (see
 Required packages:
 - C++ compiler with C++17 support
 - [CMake](https://cmake.org/) >= 3.14.0
+- clang-format >= 15 for code formatting
```

Comment:
I'd make this package optional (not under "required") and with clang-format it's usually safer to use the precise version (equal), not `>=`

To copy-paste from UR:
```
For development and contributions:
- clang-format-15.0 (can be installed with `python -m pip install clang-format==15.0.7`)
```

Reply:
done